### PR TITLE
New package: jeti-filemanager-1.1.2

### PR DESCRIPTION
**Pull request by mrshampoo**
jeti-filemanager take 15 ;)

### CODE REVIEW

**Comment by mrshampoo** [comment]
sorry i didn't make a  new pull request on purpose, i wanted to reopen my last., i was just a bit unsure of how i did it last time..

File: srcpkgs/jeti-filemanager/template [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Template file for 'jeti-filemanager'
+pkgname=jeti-filemanager
+version=1.1.2
+revision=1
+wrksrc=jeti-filemanager-${version}
```
Comment (pullmoll) [inline srcpkgs/jeti-filemanager/template]: This line is redundant, because `wrksrc=${pkgname}-${version}` is the default. Remove it.

**Comment by mrshampoo** [comment]
fuck what did i do wrong now?!??!

**Comment by mrshampoo** [comment]
omg :O

**Comment by mrshampoo** [comment]
i really hate this shit!

**Comment by Vaelatern** [comment]
`git rebase master`

**Comment by mrshampoo** [comment]
says nothing to me.. i  rm the whole dir and start all over

**Pull request closed without merging**